spec/javascripts/health-care/components/personal-information/VaMedicalFacility.spec.jsx

```diff
@@ -47,4 +58,23 @@ describe('<VaMedicalFacility>', () => {
     const selects = ReactTestUtils.scryRenderedDOMComponentsWithTag(component, 'select');
     expect(selects).to.have.length(1);
   });
+
+  // TODO(crew): Once options are refactored to build blank option with data, fix these tests.
+  it('has no options if facilityState is NOT specified', () => {
+    const component = ReactTestUtils.renderIntoDocument(
+      <VaMedicalFacility value="test" facilityState="{}"/>
```

Comment:
Do you mind changing `facilityState` to be an empty string instead? That might be a more accurate representation of what it would equal if no state is selected.